src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,52 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Item {
+  title: string,
+  description?: string,
+  imgUrl: string,
+  imdbUrl: string,
+}
```

Comment:
I think you can move this interface to seperate file and just imort this when you neded